src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/tasks/Task.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.tasks
+
+import net.javacrumbs.shedlock.core.LockAssert
+import org.slf4j.LoggerFactory
+import uk.gov.justice.digital.hmpps.pecs.jpc.service.MonitoringService
+
+abstract class Task(private val name: String, private val monitoringService: MonitoringService) {
+
+  private val logger = LoggerFactory.getLogger(javaClass)
+
+  fun execute() {
+    Result.runCatching {
+      LockAssert.assertLocked()
```

Comment:
So reading the docs a bit more on this this should help prevent configuration errors.  I have also take the opportunity to wrap some monitoring around so it won't just silently fail/log.